flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/row/ParquetRowDataWriter.java

```diff
@@ -165,6 +187,11 @@ public void write(RowData row, int ordinal) {
         public void write(RowData row, int ordinal) {
             recordConsumer.addLong(row.getLong(ordinal));
         }
+
+        @Override
+        public void write(ArrayData arrayData, int ordinal) {
```

Comment:
It is better to add a method:
```
private void writeLong(long value) {
   recordConsumer.addLong(value);
}
```
Then these two method can reuse logical.

Reply:
done